stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/common/avatar/UserAvatar.kt

```diff
@@ -0,0 +1,44 @@
+package io.getstream.chat.android.compose.ui.common.avatar
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Shape
+import coil.compose.rememberImagePainter
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.models.image
+import io.getstream.chat.android.client.models.initials
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Represents the [User] avatar that's shown on the Messages screen or in headers of DMs.
+ *
+ * Based on the state within the [User], we either show an image or their initials.
+ *
+ * @param user - The user whose avatars we want to show.
+ * @param modifier - Modifier for styling.
+ * @param shape - The shape of the avatar.
+ * @param contentDescription - The content description of the avatar.
+ * @param onClick - The handler when the user clicks on the avatar.
+ * */
+@Composable
+public fun UserAvatar(
+    user: User,
+    modifier: Modifier = Modifier,
+    shape: Shape = ChatTheme.shapes.avatar,
+    contentDescription: String? = null,
```

Comment:
Would it make sense to default this to `user.name`, or is that nonsense in some cases?

Reply:
I think it's better to be null, as we shouldn't fill the accessibility data with names of users - screen readers should be able to catch the text elements anyway.

When it's used in channel lists, when we have 1:1 DMs, people will likely use SRs to read the "channel name" label anyway. This would be a description on an image that doesn't give much info.